corehq/apps/export/views/new.py

```diff
@@ -177,9 +177,12 @@ def format_options(self):
         format_options = ["xls", "xlsx", "csv"]
 
         should_support_geojson = (
-            self.export_type == CASE_EXPORT
-            and toggles.SUPPORT_GEO_JSON_EXPORT.enabled(self.domain)
-            and not self._is_bulk_export
+            toggles.SUPPORT_GEO_JSON_EXPORT.enabled(self.domain)
+            and (
```

Comment:
Nit: Instead of having a separate check for both case and form exports, this can be shortened to `self.export_type in [CASE_EXPORT, FORM_EXPORT]`

Reply:
@zandre-eng I also need to consider the `self._is_bulk_export`. I changed the flow a little [here](https://github.com/dimagi/commcare-hq/pull/34281/commits/2be9d50a810272c8007cfda7ad83fae81677c7fc).

Reply:
Thanks, that looks good as well. :+1: